pkg/envinfo/fake.go

```diff
@@ -0,0 +1,7 @@
+package envinfo
+
+func GetFakeEnvInfo(settings ComponentSettings) *EnvInfo {
```

Comment:
Can you please add comment what does this function do?

Reply:
we keep fake stuff in _test.go files as they are for testing perposes if I am not wrong?

Reply:
I tried that but doesn't work. I think we cannot use functions from _test.go in the main code. Also found this article https://stackoverflow.com/questions/59064256/is-it-possible-to-call-a-test-func-from-another-file-to-start-the-testing